### PR TITLE
[WASM] Missing signatures for OpenGLES

### LINKED ISSUE #19612: [WASM] CANNOT HANDLE COOKIE VFF
Hi, @MarcosCobena and I are trying to create a OpenGLES[1,2,3,4] binding for mono wasm. We have successfully compiled the emscripten OpenGLES libraries in C++ but, when calling some of them from .Net using P/Invoke, we encounter the `CANNOT HANDLE COOKIE` error because some signatures are missing in the [pre-defined signature list](https://github.com/mono/mono/blob/6c0bfdc3f3d5855d27628112a505ba01bdfc4584/mono/mini/m2n-gen.cs#L30)

## Steps to Reproduce

1. Create a library that exports a function with signature VFF like `void dummy(float one, float two)`
2. Compile that library using emscripten
3. Create a WebAssembly mono app that calls that function thorugh P/Invoke.
4. Compile the application linking the first library
5. Run the app on a browser

### Current Behavior

Browser console throws `CANNOT HANDLE COOKIE VFF` and function is not executed.

### Expected Behavior

Function is executed normally.

## On which platforms did you notice this

[x] Firefox
[x] Chrome
Not tested on other browsers

**Version Used**:

Mono master:

commit d525e3d
Author: Kenneth Pouncey kjpou@pt.lu
Date: Tue Mar 31 06:33:30 2020 +0200

## Info

The missing signatures that we need for OpenGLES are:

```
IIIL
VF
VFF
VFFF
VFFFF
VFFFFFF
VFFFFFFFF
VFI
VIFFF
VIIF
VIIFF
VIIFFF
VIIFFFF
VIIFI
VIIIIIIIIIIIIIII
VIIL
```

### PR DESCRIPTION
**Pull request by emepetres**
Fixes #19612

Added missing signatures for OpenGLES binding in WebAssembly.

### CODE REVIEW

File: mono/mini/m2n-gen.cs [hunk 1]
```diff
@@ -105,13 +107,28 @@ class Driver {
...
 		"VIFFFF",
 		"VIFFFFF",
 		"VIFFFFFF",
 		"VIFFFFFI",
+		"VIIF",
+		"VIIFF",
+		"VIIFFF",
+		"VIIFFFF",
 		"VIIFFI",
+		"VIIFI"
```
Comment (lewing) [inline mono/mini/m2n-gen.cs:131]:
```suggestion
		"VIIFI",
```

**Comment by kjpou1** [comment]
@monojenkins build

**Comment by kjpou1** [comment]
@monojenkins build

**Comment by emepetres** [comment]
Closing as it is not valid anymore. A new PR will be done.

**Pull request closed without merging**